src/bio.c

```diff
@@ -217,7 +227,7 @@ void *bioProcessBackgroundJobs(void *arg) {
         /* Process the job accordingly to its type. */
         if (type == BIO_CLOSE_FILE) {
             close(job->fd);
-        } else if (type == BIO_AOF_FSYNC) {
+        } else if (type == BIO_AOF_FSYNC || type == BIO_FSYNC_AND_CLOSE) {
```

Comment:
i'm not certain if we want all the logic here (aof_bio_fsync_status) when syncing the previous file.
if we do, it means the new job type must be part of BIO_AOF_FSYNC (with an additional close), if we don't then it could be a variant of BIO_CLOSE_FILE.

arguably there's no much good in setting aof_bio_fsync_status since the next fsync attempt will be done on the new file and if successful, will reset that variable anyway.

Reply:
> arguably there's no much good in setting aof_bio_fsync_status since the next fsync attempt will be done on the new file and if successful, will reset that variable anyway.

yes, make sense, i dropped it